pkg/reconciler/timeout_handler.go

```diff
@@ -170,7 +170,6 @@ func backoffDuration(count uint, jf jitterFunc) time.Duration {
 func (t *TimeoutSet) checkPipelineRunTimeouts(namespace string, pipelineclientset clientset.Interface) {
 	pipelineRuns, err := pipelineclientset.TektonV1alpha1().PipelineRuns(namespace).List(metav1.ListOptions{})
 	if err != nil {
-		t.logger.Errorf("Can't get pipelinerun list in namespace %s: %s", namespace, err)
```

Comment:
I'm not super-happy to remove logging just ahead of a release - it may make troubleshooting harder for folks. That said, removing non-thread safe behavior is probably more important, so I'm ok with this - and we still have a chance to bring logging back before the release anyways.